web-client/src/views/StartCaseInternal/StartCaseInternal.tsx

```diff
@@ -77,7 +75,7 @@ export const StartCaseInternal = connect(
                   >
                     <div className="blue-container">
                       <IRSNotice
-                        validateFormData={validateCaseDetailSequence}
+                        validateFormData={validatePetitionFromPaperSequence}
```

Comment:
We're using this everywhere else in this component, and I saw new good reason for the mismatch. Instead, it led to errors not syncing correctly in the ErrorNotification banner,